Lib/ufoLib/glifLib.py

```diff
@@ -789,16 +787,12 @@ def validateLayerInfoVersion3Data(infoData):
 	a set range of possible values for an attribute, that the
 	value is in the accepted range.
 	"""
-	validInfoData = {}
-	for attr, value in list(infoData.items()):
+	for attr, value in infoData.items():
 		if attr not in layerInfoVersion3ValueData:
 			raise GlifLibError("Unknown attribute %s." % attr)
 		isValidValue = validateLayerInfoVersion3ValueForAttribute(attr, value)
 		if not isValidValue:
 			raise GlifLibError("Invalid value for attribute %s (%s)." % (attr, repr(value)))
-		else:
-			validInfoData[attr] = value
-	return validInfoData
```

Comment:
why? you spare a microsecond and you break other people's scripts?

Reply:
you forgot to restore this one?

Reply:
See the next line "return infoData"

Reply:
Ah ok, I missed that